src/reset.css

```diff
@@ -0,0 +1,11 @@
+body {
+  margin: 0;
+  width: 200px;
+}
+
+.card__title,
+.card__price,
+.card__price_value,
+.card__reviews {
```

Comment:
use tag selectors here (p, ul, for example, or any else, that you need)
don't use elements classes in reset.css file